### DISCUSSION TITLE
osu lazer not launching after I download a skin

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#16736] [Q&A]
The game works fine with the default skin but as soon as I download any skin it doesn't launch it just has the loading symbol above my cursor and on task manager it says suspended and then closes. Also regular osu works fine.

Here are the logs:

[database.log](https://github.com/ppy/osu/files/7982762/database.log)
[performance.log](https://github.com/ppy/osu/files/7982763/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7982764/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7982765/updater.log)
.

**Comment by peppy** [comment]
It looks like you have a very old version of lazer installed which has taken file extension associations.

You can try dragging the skin into the window instead if you can't figure how to fix your file associations.

**Reply by ghost** [reply]
What do you mean by dragging the skin into the window and how do I install the latest version because when I launch it it says I'm running the latest version and I've reinstalled the game multiple times?

**Reply by ghost** [reply]
Also the game started randomly crashing or completely freezing my pc and making it so I need to shutdown to get off the black screen.

**Reply by peppy** [reply]
Just uninstall it I think.

**Reply by ghost** [reply]
?

**Reply by frenzibyte** [reply]
Uninstall the game, then try reinstalling the latest version (https://github.com/ppy/osu/releases/tag/2022.128.0) and see if that makes it work for you.
